web/src/pages/auth/Login/Login.tsx

```diff
@@ -88,6 +90,7 @@ export const Login = () => {
         );
       } else {
         console.error(error);
+        toaster.error(LL.form.error.tooManyBadLoginAttemps());
       }
```

Comment:
The too many bad logins attempts error should be displayed only when the error status code is 429, so an additional else if branch may be needed

Also there is a typo:
```suggestion
      } else {
        console.error(error);
        toaster.error(LL.form.error.tooManyBadLoginAttempts());
      }
```